documentation/modules/kafkaconnectors/proc-deploying-kafkaconnector.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-connect.adoc
+
+[id='proc-deploying-kafkaconnector-{context}']
+= Deploying a `KafkaConnector` resource to Kafka Connect
+
+Deploy the example `KafkaConnector` to a Kafka Connect cluster. The example YAML file is configured to create a `FileStreamSourceConnector` to send each line of the Kafka license file as a message in Kafka.
+
+.Prerequisites
+
+* A Kafka Connect deployment in which xref:proc-enabling-kafkaconnectors-deployment-configuration-kafka-connect[`KafkaConnectors` are enabled]
+* A running Cluster Operator
+
+.Procedure
+
+. Edit the `examples/connector/source-connector.yaml` file:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: `KafkaConnector`
+metadata:
+  name: my-source-connector <1>
+  labels:   
+    strimzi.io/cluster: my-connect-cluster <2>
+spec:
+  class: org.apache.kafka.connect.file.FileStreamSourceConnector <3>
+  tasksMax: 2 <4>
+  config: <5>
+    file: "/opt/kafka/LICENSE"
+    topic: my-topic
+    # ...
+----
++
+<1> Enter a name for the `KafkaConnector` resource. You can choose any name.
```

Comment:
```suggestion
<1> Enter a name for the `KafkaConnector` resource. This will also be the name of the connector within Kafka Connect. You can choose any name that's valid for a Kubernetes resource.
```